docs/UserGuide.md

```diff
@@ -218,9 +229,23 @@ _Details coming soon ..._
 
 ## FAQ
 
+Listed here are some frequently asked questions. Should you require any other assistance, feel free to [contact us](#contact-us).
+
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous GuestBook home folder.
 
+**Q**: Will my data be automatically saved?<br>
+**A**: Yes, Guest Book automatically saves your data after every command entered.
+
+**Q**: Where is my data saved?<br>
+**A**: Go to where you have downloaded your `guestbook.jar` file, there a `/data` folder will be created and the data will be saved under `guestbook.json`.
+
+**Q**: Can I rename my saved data file?<br>
+**A**: No, Guest Book currently only supports the use of `guestbook.json` as the name of the saved data file.
+
+**Q**: Why are there no guests in my Guest Book?<br>
+**A**: If you didn't enter the `clear` command, it is likely that your `guestbook.json` file is corrupted, causing Guest Book to reset it to prevent any malicious files from being used.
+
```

Comment:
Should we add a FAQ about what to do to seek help? And perhaps direct to the Help button

Reply:
But the help button leads to the UserGuide which is where they are at?

Reply:
Oh, that is true!